DB.py

```diff
@@ -31,26 +31,55 @@ def addq(ctx, member: discord.Member, *, quote: str):
 
 @bot.command()
 @asyncio.coroutine
-def q(member: discord.Member, *, query: str=None):
+def q(member: discord.Member=None, *, query: str=None):
 	conn = sqlite3.connect(DB_PATH)
 	c = conn.cursor()
-	t = (member.id,)
-	if query is None:
-		quoteslist = c.execute('SELECT Quote FROM Quotes WHERE ID=?',t).fetchall()
+	if member is None:
+		quotes = c.execute('SELECT Quote FROM Quotes').fetchall()
+		quote = random.choice(quotes)[0]
+		ID = c.execute('SELECT ID FROM Quotes WHERE Quote LIKE "%%%s%%"' % quote).fetchall()[0][0]
+		yield from bot.say("<@%(ID)s> :mega: %(quote)s" % {"ID": ID, "quote": quote})
+		conn.close
```

Comment:
this is the case when no member argument was supplied, and it gets a random quote from the entire database, and says it while pinging that who said it